src/main/java/com/project/policyNews/controller/UserController.java

```diff
@@ -0,0 +1,60 @@
+package com.project.policyNews.controller;
+
+import com.project.policyNews.entity.User;
+import com.project.policyNews.model.Auth;
+import com.project.policyNews.security.JwtTokenProvider;
+import com.project.policyNews.service.UserService;
+import java.util.Map;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@Slf4j
+@RestController
+@RequestMapping("/auth")
+@RequiredArgsConstructor
+public class UserController {
+
+  private static final Logger logger = LoggerFactory.getLogger(UserController.class);
+
+  final private UserService userService;
+
+  final private JwtTokenProvider jwtTokenProvider;
+
+  // 회원가입
+  @PostMapping("/register")
+  public ResponseEntity<?> register(@RequestBody Auth.register user) {
+
+    if (userService.userExists(user.getUsername())) {
+      logger.warn("이미 회원가입된 ID 입니다.");
+      return ResponseEntity.badRequest().body("이미 회원가입된 ID 입니다.");
+    }
+    if (user.getRoles() == null || user.getRoles().isEmpty()) {
+      user.getRoles().add("USER");
+    }
+    User newUser = userService.registerUser(user);
+
+    logger.info("회원가입이 성공적으로 처리되었습니다. [" + user.getUsername() +"]");
```

Comment:
+로 하지 않고 {}를 사용하시면 좋을 것 같아요
```suggestion
    logger.info("회원가입이 성공적으로 처리되었습니다. [{}]", user.getUsername());
```